README.md

```diff
@@ -1,4 +1,5 @@
-# truex-tvos-integrations
-Documentation and reference apps for TrueX's tvOS based integration library.
+# true[X] tvOS Integrations
```

Comment:
Nice polish! 

Maybe we should link the reference app on the main README page too? Thoughts?

Reply:
Yes, good idea.